### ISSUE TITLE
ENROLL function is broken

### ISSUE THREAD

**Issue by NicFragale** [openziti-incubator/zssh#20]
Specifying "./zssh enroll XXX.jwt" appears to work initially, but fails shortly after with the usage statement of the program.  Enrollment does not complete and does not produce a JSON configuration.

**Comment by dovholuknf** [comment]
I will try on macos x86 but on windows it completed fine:
```
zssh-windows-amd64.exe enroll c:\temp\aa.jwt
INFO    generating 4096 bit RSA key
INFO    enrolled successfully. identity file written to: c:\temp\aa.json
```

**Comment by dovholuknf** [comment]
works on mac amd x86/x64 too...  Could the jwt have been invalid/old?
```
➜  ~ ./zssh-macos-amd64 enroll macaa.jwt
INFO    generating 4096 bit RSA key
INFO    enrolled successfully. identity file written to: macaa.json
```

**Comment by dovholuknf** [comment]
I pushed v0.0.13 out https://github.com/openziti-test-kitchen/zssh/releases/tag/0.0.13

![image](https://user-images.githubusercontent.com/46322585/169895216-17a18856-ba33-4fc5-b45b-02a163a8d2c9.png)

You'll now see 'why' enrollment failed.

**Comment by NicFragale** [comment]
@dovholuknf This illuminated the issue Clint!  

For reference.  I recreated the JWT.  Without modifying it, I referenced it in the enroll process again and it produced the following.  Now I need to understand why it is thinking its not a standards compliant certificate when generated by MOP.

ERROR   failed to enroll: Post "....fc4008ca.production.netfoundry.io:443/enroll?method=ott&token=....": x509: “150.136.233.219” certificate is not standards compliant
